src/components/leaderboards/TeamLeaderboardItem/__tests__/TeamLeaderboardItem-test.js

```diff
@@ -30,4 +30,18 @@ describe('TeamLeaderboardItem', function() {
       expect(elementName).not.toBeNull();
     });
   });
+
+  describe('Render charity name option', function() {
+    var leaderboardItem;
+    var element;
+    var elementCharity;
+
+    it('renders a charity name if present', function() {
+      teamLeaderboardItem = <TeamLeaderboardItem charityName='foo' />;
```

Comment:
Super small! The variable here is declared above as leaderboardItem (probably want to declare it as teamLeaderboardItem I guess)